.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+services:
+    - postgresql
+install:
+  - pip install -e . -r requirements-dev.txt
+
+before_script:
+  # add roles pgbedrock needs
+  - psql -c 'create database test_db;' -U postgres
```

Comment:
[nitpick] Can we capitalize keywords in all statements here? e.g. CREATE ROLE, PASSWORD, SUPERUSER, etc.

Reply:
[super nitpick] Also: your use of single and double quotes is inconsistent throughout this file.